ui/shared/status/buttons/StatusIconTabButton.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+import "../../../imports"
+import "../../"
+import "../core"
+import "../"
+
+TabButton {
+    id: control
+
+    property color iconColor: Style.current.secondaryText
+    property color disabledColor: iconColor
+    property string name: ""
+
+    implicitWidth: 40
+    implicitHeight: 40
+
+    icon.height: 24
+    icon.width: 24
+    icon.color: {
+        if (!enabled) {
+            return control.disabledColor
+        }
+        return (hovered || checked) ? Style.current.blue : control.iconColor
+    }
+
+    contentItem: Item {
+        anchors.fill: parent
+
+        Loader {
+            active: true
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors.verticalCenter: parent.verticalCenter
+            sourceComponent: control.name !== "" ? letterIdenticon :
+              !!icon.source.toString() ? imageIcon : defaultIcon
+        }
+
+        Component {
+            id: defaultIcon
+            StatusIcon {
+                icon: control.icon.name
+                height: control.icon.height
+                width: control.icon.width
+                color: control.icon.color
+            }
+        }
+
+        Component {
+            id: imageIcon
+            RoundedImage {
+                source: icon.source
+                noMouseArea: true
+            }
```

Comment:
This `RoundedImage` component most likely needs to be revisited. Don't even know why it has a `noMouseArea` property.